syntaxes/crystal.json

```diff
@@ -129,15 +129,20 @@
 			"name": "meta.lib.type.crystal"
 		},
 		{
-			"comment": "everything being a reserved word, not a value and needing a 'end' is a..",
+			"comment": "everything being a reserved word, not a value, and needing a 'end' is a..",
 			"match": "(?<!\\.)\\b(fun|begin|case|class|else|elsif|end|ensure|enum|for|if|macro|module|rescue|struct|then|union|unless|until|when|while)\\b(?![?!:])",
 			"name": "keyword.control.crystal"
 		},
 		{
-			"comment": "everything being a reserved word, not a value and not needing a 'end' is a..",
+			"comment": "everything being a reserved word, not a value, and not needing a 'end' is a..",
 			"match": "(?<!\\.)\\b(abstract|alias|asm|break|extend|in|include|next|of|private|protected|struct|return|select|super|with|yield)\\b(?![?!:])",
 			"name": "keyword.control.primary.crystal"
 		},
+		{
+			"comment": "everything being a spec keyword, not a value, and needing a block is a..",
+			"match": "(?<!\\.)\\b(describe|context|it|expect_raises)\\b(?![?!:])",
+			"name": "keyword.control.crystal"
+		},
```

Comment:
Was it intentional not to mark every spec related methods like `should`, `be_true`, `eq` etc as a "method" equivalent ?

Reply:
Yes, I didn't think it was necessary and potentially cause for confusion as those method names are likely to exist outside of the spec context. I also wanted the spec highlighting to be consistent with other highlighting (i.e., there aren't any other method calls that are highlighted without parenthesis)